acceptance/testdata/restorer/container/to_cache/unused_buildpack/layers/unused_buildpack/cached-layer/data

```diff
@@ -0,0 +1 @@
+cached-data
```

Comment:
Maybe it's better to change this a bit to something different than the content of `acceptance/testdata/restorer/container/to_cache/cacher_buildpack/layers/cacher_buildpack/cached-layer/data` so the code will be more readable.
What do you think?